src/main/java/spoon/support/reflect/code/CtAssertImpl.java

```diff
@@ -46,7 +46,8 @@ public void accept(CtVisitor visitor) {
 	}
 
 	public void setExpression(CtExpression<T> value) {
-		value.setParent(this);
+		if (value != null)
+			value.setParent(this);
```

Comment:
An expression of an assertion can be null? For me, an assertion must have an expression so I propose to change for something like:

```java
public void setExpression(CtExpression<T> value) {
    if (value == null) {
        throw new SpoonException("An assertion must have an expression.");
    }
    value.setParent(this);
    this.value = value;
}
```

What do you think?